json-csharp/IteratingOverJsonObject/IteratingOverJsonObject/TestData.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace IteratingOverJsonObject;
+
+public class TestData
+{
+    public string GenerateJsonData()
+    {
+        var jsonData = @"[
+            {
+                ""name"":""John Doe"",
+                ""age"": 30,
+                ""department"":""IT""
+            },
+            {
+                ""name"":""Antonio Weber"",
+                ""age"": 20,
+                ""department"":""Finance""
+            },
+            {
+                ""name"":""Jane Doe"",
+                ""age"": 25,
+                ""department"":""Legal""
+            },
+            {
+                ""name"":""Ramiro Buck"",
+                ""age"": 28,
+                ""department"":""IT""
+            },
+            {
+                ""name"":""Shirley Pearson"",
+                ""age"": 30,
+                ""department"":""Human Resource""
+            }
+        ]";
```

Comment:
We can use a string literal with triple quotes (""") that was introduced in .Net 7 for better readability.